PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java

```diff
@@ -69,19 +69,19 @@
     // Shared Preferences
     private SharedPreferences mUserPreferences;
     private ProgressDialog mProgressDialog;
-    /** Parent Activity **/
+    // Parent Activity *
```

Comment:
also i noticed that we seem to be using opposite conventions for documentations.
@chenhoward preferred the /** **/ for documenting class variables, and the "//" for in-line comments within methods.